components/organisms/messaging/components/audioPlayer/wrapper.js

```diff
@@ -0,0 +1,276 @@
+import styled, { css } from 'styled-components'
+
+export const AudioWrapper = styled.div`
+  width: 100%;
+  ${({ hidden }) =>
+    hidden === true &&
+    css`
+      display: none;
+    `}
+  ${({ preview }) =>
+    preview &&
+    css`
+      background-color: ${({ theme: { MESSAGING } }) => MESSAGING.INPUT_SECTION_BACKGROUND_COLOUR};
+      display: flex;
+      left: 4rem;
+      position: absolute;
+      width: calc(100% - 8rem);
+      z-index: 2;
+    `}
+.rhap_container {
```

Comment:
Lots of colors in this styles that can use theme. Let's use theme for them as much as possible.